back/spec/jobs/delete_invites_job_spec.rb

```diff
@@ -26,6 +26,23 @@
 
     it 'deletes only expired invites' do
       described_class.perform_now(expiry_time)
+
+      expect(invite1.reload).to eq invite1
+      expect { invite2.reload }.to raise_error(ActiveRecord::RecordNotFound)
+      expect { invite3.reload }.to raise_error(ActiveRecord::RecordNotFound)
+      expect(invite4.reload).to eq invite4
+    end
+
+    it 'deletes records correctly and w/o errors when two jobs are called in parallel', use_transactional_fixtures: false do
+      expect(ErrorReporter).not_to receive(:report)
```

Comment:
This test fails 10 times out of 10 without the `includes` fix.

Reply:
Very nice! 👍